builder_road.nut

```diff
@@ -33,6 +33,7 @@ class BuildRoad extends Task {
 	
 	function FindPath(a, b) {
 		local pathfinder = Road();
+		pathfinder._pathfinder._queue_class = AIPriorityQueue;

```

Comment:
Nice one! I just happened to find out about that yesterday, but I still had it wrapped in a Squirrel class, this is even cleaner.